Extension/CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.29.0-insiders: June 24, 2020
+### New Features
+* Add Doxygen comment support (to tooltip display of hover, completion, and signature help). [#658](https://github.com/microsoft/vscode-cpptools/issues/658)
+
+### Enhancements
+* Add support for the Debug Welcome Panel. [#4837](https://github.com/microsoft/vscode-cpptools/issues/4837)
+* Update to clang-format 10. [#5194](https://github.com/microsoft/vscode-cpptools/issues/5194)
+* Added system to store and query properties from the active C/C++ configuration.
+  * bugengine (@bugengine) [PR #5453](https://github.com/microsoft/vscode-cpptools/pull/5453)
+* Add 'quoteArgs` to `launch.json` schema. [PR #5639](https://github.com/microsoft/vscode-cpptools/pull/5639)
```

Comment:
I made a mistake here, theres a ' instead of a ` around _quoteArgs_

Reply:
ok